packages/dev-portal/src/config.ts

```diff
@@ -48,7 +48,15 @@ export const configure = async (config: FrameworkConfigurator) => {
 
   enableAppModule(config);
 
-  enableNavigation(config);
+  enableNavigation(config, {
+    configure: (config) => {
```

Comment:
missing  `config.setBasename()` and `config.setHistory(createHistory('browser'))`?